docs/testing.md

```diff
@@ -169,8 +162,9 @@ which checks the JavaScript against the rules configured in `.eslintrc`.
 for detailed rule descriptions.
 
 There are a number of options to the command:
- - Use `gulp lint:build` to only lint the build scripts.
- - Use `gulp lint:test` to only lint the test scripts.
- - Use `gulp lint:scripts` to only lint the project source scripts.
- - Add the `--fix` flag (like `gulp lint --fix`) to auto-fix
+
+ - `gulp lint:build`: Lint only the gulp build scripts.
+ - `gulp lint:test`: Lint only the test scripts.
+ - `gulp lint:scripts`: Lint only the project source scripts.
+ - `--fix`: Add this flag (like `gulp lint --fix`) to auto-fix
```

Comment:
> Why not autofix all the files?

I guess the same reason why someone would choose to only lint test or build scripts instead of just running `gulp lint`. ¯\\_(ツ)_/¯